packages/neon-core/src/tx/transaction/PublishTransaction.ts

```diff
@@ -0,0 +1,195 @@
+import { TX_VERSION } from "../../consts";
+import {
+  ab2hexstring,
+  hexstring2str,
+  num2hexstring,
+  num2VarInt,
+  str2hexstring,
+  StringStream
+} from "../../u";
+import { BaseTransaction, TransactionLike } from "./BaseTransaction";
+import TransactionType from "./TransactionType";
+
+export enum ContractParameterType {
```

Comment:
This is defined at `sc.ContractParamType`.

Reply:
Thanks for the hint. I will leverage whatever we already have.